packages/connect-thegraph-tokens/subgraph/manifest/templates/contracts/Kernel.template.yaml

```diff
@@ -39,4 +39,6 @@
     eventHandlers:
       - event: NewAppProxy(address,bool,bytes32)
         handler: handleNewAppProxy
+    blockHandlers:
```

Comment:
Kernel data sources now trigger actions on new blocks. Yay!

This should work on all networks, and we need to monitor if this makes indexing slower. I don't think it should.

Reply:
Trying to understand this mechanism a bit better—we are essentially asking to be notified on each new block that we see?

We place this handler on the kernel because we have no where better to put it?

Reply:
Looks like it is making things slower. Needs optimization.

Reply:
After discussing with The Graph, there doesn't seem to be a way to accelerate block handling. Even after reducing the amount of triggers that cause a block to be processed, its still very slow. So, unfortunately, we'll have to forego this option.